extensions/adobe/experience/offer-management/representation-content.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/representation-content",
+  "type": "object",
+  "title": "Content",
+  "description": "Offer representation content",
```

Comment:
This is insufficient as a description.

Reply:
Representations as a part of the offer model are now well explained in offer and placement documentation.